### PR TITLE
cigar case resprite

### PR DESCRIPTION
**Pull request by CeUvi**
## About The Pull Request
!!CONFLICTS WITH #6804, DON'T MERGE #6804!!
image
cigar case resprite, locked+open states, needs to be open to take a cigar, storage capacity reduced from 7 to 6(not nerf, for the sake of the sprite), sprites by me and InfraredBaron
mini-cigars are approved by BARON himself
making this because kegdo is stubborn and doesn't listen to me and Baron.

p.s.: proof of baron's approval 
image
image
(go learn russian)
he says: 
- make a comparsion
- hmm
- small cigars look good

p.p.s.: 
image
we did it, gamers
he chose minicigars
## Why It's Good For The Game
bay leftovers bad
## Changelog
:cl: CeUvi, Kegdo
add: cigar case requires opening before taking the cigars out
tweak: cigar case now holds only 6 cigars instead of 7
imageadd: cigar case resprite
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
oh my fucking god

**Comment by CeUvi** [comment]
i told you

**Comment by Kegdo** [comment]
conflicts with #6804 and with common sense, ceuvi i swear to god

**Comment by Kegdo** [comment]
And I am also fairly sure some maintainer will not check comments or anything and then merge this and get a conflict

**Comment by Kegdo** [comment]
image

**Comment by CeUvi** [comment]
сам виноват.

**Comment by SirRichardFrancis** [comment]
Sprite is great, but already introduced in https://github.com/discordia-space/CEV-Eris/pull/6804.

**Comment by CeUvi** [comment]
introduced only because kegdo doesn't listen to people

**Comment by SirRichardFrancis** [comment]
> kegdo doesn't listen to people

Neither do you, smh.

image

**Comment by CeUvi** [comment]
I listen to Baron tho, while kegdo doesn't.

**Comment by Kegdo** [comment]
A simple reason - the smol tiny wee wee cigars are bad

**Comment by Kegdo** [comment]
And you can't prove me wrong

**Comment by CeUvi** [comment]
> A simple reason - the smol tiny wee wee cigars are bad

post ingame pictures, let people decide.

**Comment by Kegdo** [comment]
> > A simple reason - the smol tiny wee wee cigars are bad
> 
> post ingame pictures, let people decide.

image
^yours(bad)

image
^yours(good)

**Comment by CeUvi** [comment]
how did you fit this fucking PIPE 
image
inside of this small case 
image

**Comment by Kegdo** [comment]
with the same logic you can say that a monkey is the size of two zippos

the cigars are bigger so the players won't have to pixel hunt to ignite them

**Comment by CeUvi** [comment]
> the cigars are bigger so the players won't have to pixel hunt to ignite them

go playtest it with real people then, I want to hear actual feedback

**Comment by SirRichardFrancis** [comment]
> how did you fit this fucking PIPE 

image

Dude, we have ammo that look huge when outside of magazine, but tiny when inside. And the same story with many things.

image

Your cigar too looks bigger than a human torso, but it's not a real size, is it?

**Comment by Kegdo** [comment]
obliterated with facts and logic. Now close your pr

**Comment by CeUvi** [comment]
> Dude, we have ammo that look huge when outside of magazine, but tiny when inside. And the same story with many things.

ammo is the kind of thing that you need to access quickly most of the time.
cigars aren't that kind of thing, especially with opening and closing cases addition.



> Now close your pr

nope, cry about it.

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -286,18 +286,83 @@
...
 	w_class = ITEM_SIZE_TINY
 	throwforce = WEAPON_FORCE_HARMLESS
 	slot_flags = SLOT_BELT
-	storage_slots = 7
+	storage_slots = 6
 	can_hold = list(/obj/item/clothing/mask/smokable/cigarette/cigar)
 	icon_type = "cigar"
 	reagent_flags = REFILLABLE | NO_REACT
 	item_obj = /obj/item/clothing/mask/smokable/cigarette/cigar
+	var/open = FALSE
```
Comment (Kegdo) [inline code/game/objects/items/weapons/storage/fancy.dm:301]: shamelessly stole my code!! gitban he!

Comment (CeUvi) [reply code/game/objects/items/weapons/storage/fancy.dm:301]: cry about it

**Comment by Kurgis** [comment]
Using different world and inventory icons would solve both of your issues, but I don't believe Eris has that implemented as of yet.

**Pull request closed without merging**